### ISSUE TITLE
Disable discard

### ISSUE THREAD

**Issue by coolmyll** [Nightonke/BoomMenu#137]
Is there any way to disable discard of the menu? 
I want to force a choice.

**Comment by ziadkiwan** [comment]
yep there is something called bmb.setiscancelabed if i'm not mistaken!
